ai-decision-services/ai-ad-dpp-engine/setup-ad/setup-anomaly-detection.md

```diff
@@ -0,0 +1,10 @@
+Lab 1: Setup Anomaly Detection
+=================
+
+Here we will use AD as the example of post processing. Go to AI Services → Anomaly Detection, 
+
+![](./images/Set-AD1.png)
+and create a project there.
+![](./images/Set-AD2.png)
+
+Take down the OCID of your project, which will be used below.
```

Comment:
Consider rephrasing to -
Note/Take down the OCID of the AD project.  This OCID will be used in subsequent (/ next) Labs.